profile/src/main/java/rdx/works/profile/domain/GetProfileUseCase.kt

```diff
@@ -36,6 +37,9 @@ class GetProfileUseCase @Inject constructor(private val profileRepository: Profi
 /**
  * Accounts on network
  */
+val GetProfileUseCase.entitiesOnCurrentNetwork
```

Comment:
Maybe **notHiddenEntitiesOnCurrentNetwork** ?

Reply:
Good point @raf-rdx

Reply:
then we would need to rename all accounts/personas methods if we follow that convention, right now we don't do that